bouncer/full_bouncer.sh

```diff
@@ -9,6 +9,7 @@ echo "Running nightly tests 🧪"
 ./tests/rotates_through_btc_swap.ts
 ./tests/btc_utxo_consolidation.ts
 ./tests/rotation_barrier.ts
+./tests/multiple_members_governance.ts
```

Comment:
I thought it might of been interfering with the boost test, but now i think it was fine. But it still makes sense to me, A test that changes the environment like that should not run at the same time as other tests, so i moved it to full bouncer instead.